bindings/wasm/examples/index.html

```diff
@@ -48,7 +48,8 @@
         </div>
       </div>
       <button type="button" id="compile" class="margin green" disabled><span>Run</span></button>
-      <button type="button" id="download" class="blue margin">Save GLB</button>
+      <button type="button" id="glb" class="blue margin">Save GLB</button>
+      <button type="button" id="threemf" class="blue margin">Save 3MF</button>
```

Comment:
I can do this

Reply:
Oh it seems that the dropdown you are using is very manual. I thought that it would be some sort of a reusable component similar to bootstrap.

Not sure what is the cleanest way of implementing this, I guess I will leave it for another PR.